.gitignore

```diff
@@ -57,3 +57,6 @@ coverage.xml
 
 # PyBuilder
 target/
+
+# Local environment
+.env.local
```

Comment:
@JakePeyser:
I used it to replace the ERP_SERVICE  with one pointing to localhost so that I could run the tests against a modified version of the service.

export ERP_SERVICE=http://0.0.0.0:3000/api/v1/